### ISSUE TITLE
Adding awesome-cytodata to Awesome Curation

### ISSUE THREAD

**Issue by gwaybio** [cytodata/awesome-cytodata#1]
We have an option of adding our `awesome-cytodata` collection to the `awesome` stuff curation page (https://github.com/sindresorhus/awesome). 

They have quite a complicated [contribution guideline](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md)! (with over 100K stars, this is not that surprising!)

We should consider doing this. It will

1. increase visibility
2. solidify what does and what does not belong in this repo

If we decide to add to the official awesome list, I think we should restructure the repo a bit.

* Do not add publications as issues (this is a good idea that we should do anyway in a separate repo)
* Only maintain "awesome" publication resources.
  * This probably only includes comprehensive review articles.
  * And maybe some landmark papers?
* All else stays the same (and should be added upon). i.e. `data sets`, `software`, `etc.`

**Comment by fheigwer** [comment]
I like the idea. Would you go ahead and restructure the repo?

**Comment by gwaybio** [comment]
> I like the idea. Would you go ahead and restructure the repo?

Will do!

**Comment by lopaavol** [comment]
Thank you Greg for taking over this. This was the exact plan that I started working with late last year. The list wasn't yet curated enough to be added to the awesome list.

I agree that we should keep software, datasets etc. in the list as these are resources we want to share. Earlier we had discussions about having a threshold for the number of citations to publication to be accepted to the list. However, we decided to go with more qualitative selection since with threshold even a truly great paper doesn't make it to the list until about two years after publication and application papers get much more citations than landmark methodological papers.
